src/gui/base/buttons/BaseButton.ts

```diff
@@ -61,3 +64,20 @@ export class BaseButton implements ClassComponent<BaseButtonAttrs> {
 function booleanToAttributeValue(value: boolean | null | undefined): string | null {
 	return value != null ? String(value) : null
 }
+
+export function renderNewEventButton(buttonAttrs: { label: TranslationKey; click: ClickHandler; style?: Record<string, any> }): Children {
```

Comment:
The name should be more generic in this case since this button is also used to create a new email.